### DISCUSSION TITLE
CSS Theme

### DISCUSSION THREAD

**Discussion by james7342** [openstore-ecommerce/OpenStore#290] [Q&A] [answered]
Reading the guide section on Themes and Templates. Does this apply to CSS files also? I'm trying to modify the coloring of some of the elements from default. I created the /Portals/Themes/ClassicAjax/css folder structure and copied the classicajax.css file to the this new folder and adjusted a few colors. Doesn't look like css files follow the same logic as the Razor templates do. Looking at some of the code it looks like the path is hardcoded into the os_FrontOfficePageHeader.cshtml. Would be nice if this CSS followed the same logic as templates. 

https://doc.openstore-ecommerce.com/#11506

**Comment by james7342** [comment]
And the os_FrontOfficePageHeader.cshtml path is hardcoded in FrontOfficePageHeader.cshtml

**Comment by DNNMonster** [comment, marked as answer]
I think the answer is no OS does not automatically detect custom css files.

You will have to create a custom os_FrontOfficePageHeader razor file in your portal theme folder so that you can add the new link to your css.

Double check that path too...  You want to place the custom os_FrontOfficePageHeader.cshtml in  /Portals/0/Themes/ClassicAjax/default/  I am not seeing the portal number in your example.  Your css should likewise be in /Portals/0/Themes/ClassicAjax/css/

Once you do that you should be able to add links to the razor, see your css changes and be upgrade proof.  

I use custom css links exactly like this in a custom os_FrontOfficePageHeader.cshtml file :

```
<link href="/DesktopModules/NBright/NBrightBuy/Themes/ClassicAjax/css/classicajax.css" rel="stylesheet">
<link href="/Portals/0/Themes/ClassicAjax/css/classicajax.css" rel="stylesheet">
```

**Reply by james7342** [reply]
@DNNMonster, That was a type-o. I have do have the portal number in there. I had to modify 2 files and place in a my Portals/0/Themes/ClassicAjax/default/folder in order to get it to work. Along with my custom CSS file in the css folder.

FrontOfficePageHeader.cshtml
os_FrontOfficePageHeader.cshtml

![screenshot_734](https://user-images.githubusercontent.com/27960723/225402107-4dccd612-93b9-4aa1-a02e-31dbf5a2cc94.png)

**Reply by james7342** [reply]
In file FrontOfficePageHeader.cshtml I modified the below.
@RenderTemplate("\\Portals\\0\\Themes\\ClassicAjax\\Default\\os_FrontOfficePageHeader.cshtml", Model)](url)

And in file os_FrontOfficePageHeader.cshtml I modified 
`<link href="/Portals/0/Themes/ClassicAjax/css/classicajax.css" rel="stylesheet">`
